### ISSUE TITLE
[FEATURE REQUEST] Update PM2.5 definition in GEOS-Chem

### ISSUE THREAD

**Issue by yantosca** [geoschem/geos-chem#685]
The GEOS-Chem Aerosols Working Group have approved a new formulation for PM2.5, as described on the GEOS-Chem wiki: http://wiki.geos-chem.org/Particulate_matter_in_GEOS-Chem.  

This will be added into a GEOS-Chem 13.Y.0 version (TBD)

**Comment by msulprizio** [comment]
We currently don't use hardcoded growth factors but rather compute the coefficients using [the code highlighted here](https://github.com/geoschem/geos-chem/blob/b321f2f8ac3ef8d241b9db5b889e9e0b1c205576/GeosCore/aerosol_mod.F90#L367-L421) provided by Aaron van Donkelaar. We need guidance from the Aerosol WG on how to translate the new recommendations into the code.

**Comment by msulprizio** [comment]
Randall Martin wrote:
>Aaron advised that all that is needed is to change a single line (391):
```
!Set SIA growth to half of its original value to treat as partially crystalline
SIA_GROWTH = SIA_GROWTH/2e+0_fp
```

**Comment by msulprizio** [comment]
This update is now in the `dev` branch (commit https://github.com/geoschem/geos-chem/commit/ea23363909070f3877b7a96d92454533c0e585ff) and will be included in GEOS-Chem 13.1.0.

**Comment by msulprizio** [comment]
I ran a 1-hour test simulation with 13.1.0 with debug print out turned on to check the growth factor values computed in aerosol_mod.F90 using the code updates suggested by Aaron and we get:

    Growth factors at 35% RH:
     0.62 for SO4, NIT, and NH4
     1.05 for OCPI and SOA
     1.86 for SALA

That doesn't match up with the 1.10 value recommended on [the PM2.5 wiki page](http://wiki.seas.harvard.edu/geos-chem/index.php/Particulate_matter_in_GEOS-Chem#Update_in_GEOS-Chem_13.1.0).

Randall Marting wrote:

>For SIA Growth, it appears that two different definitions were being used (the value above 1, and the total value).  I think the safest option is to replace the code with the following, which is another approach that Aaron recommended. 

     !Set SIA growth above unity to half of its original value of 1.2 to treat as partially crystalline
     SIA_GROWTH = 1.1
